cmd/godmine/main.go

```diff
@@ -253,7 +258,10 @@ func createIssue() {
 	if err != nil {
 		fatal("%s\n", err)
 	}
-	c := redmine.NewClient(conf.Endpoint, conf.Apikey)
+	c, err := redmine.NewClient(conf.Endpoint, auth)
```

Comment:
🚫 **[checkstyle]** <staticcheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>SA1019: redmine.NewClient is deprecated: Use redmine.ClientBuilder to create a redmine client that supports more options and detects configuration mistakes.